README.md

```diff
@@ -1,12 +1,43 @@
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
+It uses a Python script (aka **web scraper**) that fetches details (here ‘topicals’) across all years from the Amul's website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures chronologically into your local system.
+
+### Fundamentals of Web Scraping
+Web scraping is a computer software technique of extracting information from websites. This technique mostly focuses on the transformation of unstructured data (HTML format) on the web into structured data (database or spreadsheet). <br>
+Tips:
+Get familiar with the HTML tags ; CSS, Layout & Architecture to extract the data in a better manner
+
+### Steps
+1.Import necessary libraries <br>
+2.Go to the website <br>
+3.Find the sequence in the URL of the images. Usually they are in consecutive numbers <br>
+4.Use the method of webscraping <br>
+5.Use try-catch blocks for exception might arise in between <br>
 
 ## Amul Website
 ![AMUL website screenshot](images/amul.png)
+<br>
+**Finding the Sequence :**<br>
+If you check for each year the image URL reads "http://www.amul.com/m/amul-hits?s=####" where #### represents the year number which goes way back to 1976 from 2018 (as of Dec 2018)<br>
+Try pasting this in Address Bar: http://www.amul.com/m/amul-hits?s=2018  to get redirected to the 2018 Amul Hits<br>
+
+## Zooming into each year
```

Comment:
As said above, can't keep this too because the system can change and we'd need to update our scraper itself.